From a19b8a28ff2b64509da508cfc4af17ee2ba5441b Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Thu, 18 May 2017 14:57:54 +0300
Subject: [PATCH 1995/2241] cp-110: pci-ep: Added DT bindings for PCI-EP device

Currently add a node only for the PCIe port that can be used in EP mode
in Armada-7K PCAC card and Armada-8K OCP card

Change-Id: I216ab64921f14aa6776a2e415668d403e107d71c
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43626
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
index 7f79e79..1368128 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
@@ -39,6 +39,14 @@
 		/* PCIe ports have unique addressing so they do not
 		 * come from the common CP110 dtsi
 		 */
+		pcie-ep@0x600000 {
+			compatible = "marvell,armada8k-pcie-ep", "snps,dw-pcie";
+			reg = <0 0xf2600000 0 0x4000>, <0 0xf2608000 0 0x80000>, <0 0xf2604000 0 0x1000>;
+			reg-names = "core", "lm", "shadow_core";
+			clocks = <&cpm_syscon0 1 13>;
+			status = "disabled";
+		};
+
 		pcie@0x600000 {
 			compatible = "marvell,armada8k-pcie", "snps,dw-pcie";
 			reg = <0 0xf2600000 0 0x10000>, <0 0xf6f00000 0 0x80000>;	/* Last 512KB of mem space */
-- 
2.7.4

